ironfish/src/rpc/adapters/socketAdapter/socketAdapter.ts

```diff
@@ -193,6 +197,21 @@ export abstract class RpcSocketAdapter implements IRpcAdapter {
 
       const message = result.result.data
 
+      if (this.router == null || this.router.server == null) {
+        this.emitResponse(client, this.constructUnmountedAdapter())
```

Comment:
I was wondering why this moved, but I think I understand why. You saw it doesn't respond or use the request at all, so you moved it above right? Now that this code is moving, it makes me realize that this is a small bug where it will never reject the incoming request on the client side in this case.